src/Previewer/Internal.js

```diff
@@ -189,6 +190,10 @@ export default function Internal({
     }
 
     css_provider = new Gtk.CssProvider();
+    panel_style.reset();
+    css_provider.connect("parsing-error", (self, section, error) => {
+      panel_style.handleCssParserError(section, error);
+    });
```

Comment:
@lw64 do you have thoughts on what's the best way to implement this in `updateCss` in `previewer.vala` ?

My idea would be to serialize section/error and send it on the bus but I'm not sure if it's possible for a client and how.

Reply:
It could be implemented using a dbus signal.

Reply:
:+1: 

 can you review what I did? Specially the vala part